src/app/ui/cpenvironment/process.go

```diff
@@ -108,18 +129,25 @@ func (e *Environment) showBranch0(object *dmenv.Object) {
 		imgui.SameLine()
 	}
 
+	imgui.PushStyleVarVec2(imgui.StyleVarFramePadding, imgui.Vec2{X: imgui.CurrentStyle().FramePadding().X, Y: ((e.iconSize() * (float32(e.nodesScale) / 100)) - (imgui.CalcTextSize(node.name, false, 0).Y)) / 2})
```

Comment:
Move this line to a separate func with descriptive name and use the function in places you need. Instead of copy-pasting utterly unreadable logic.. It will be also better if in the func you'll move calculations for X/Y in local vars.

Reply:
👍🏻